db/fixtures/miq_shortcuts.yml

```diff
@@ -53,208 +53,439 @@
   :description: Services / Workloads / Provision VMs
   :url: /vm_or_template/explorer?button=vm_miq_request_new
   :rbac_feature_name: vm_miq_request_new
-  :startup: false
+  :startup: true
```

Comment:
@GregP any idea why this is changed to true, i don't remember what is the use of setting startup attribute here, is that being used to ignore the shortcut entry during seeding or something. Can you please check.